docs/cli/dotnet-publish.md

```diff
@@ -0,0 +1,66 @@
+dotnet-publish
+==============
+
+## NAME
+
+`dotnet-publish` - packs the application and all of its dependencies into a folder getting it ready for publishing
+
+## SYNOPSIS
+
+dotnet-publish [--framework]  
+    [--runtime] [--output]  
+    [--configuration]  
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-publish` will compile the application, read through its dependencies specified in `project.json` and publish the resulting set of files to a directory. 
+This directory will contain the assemblies, the runtime as well as the executable version of the application. This directory can then be moved to a different machine and the application will be able to run regardless of existence of any other runtime. 
```

Comment:
This isn't quite true: you only get the runtime if you are publishing stand alone. By default you're publishing shared, so none of that happens. (You also don't get an executable.) You can only move it to another machine if it is stand alone or if the target machine has the shared framework.